content/en/docs/reference/access-authn-authz/mutating-admission-policy.md

```diff
@@ -0,0 +1,364 @@
+---
+reviewers:
+- deads2k
+- sttts
+- cici37
+title: Mutating Admission Policy
+content_type: concept
+---
+
+<!-- overview -->
+
+{{< feature-state feature_gate_name="MutatingAdmissionPolicy" >}}
+
+This page provides an overview of _MutatingAdmissionPolicies_.
+
+<!-- body -->
+
+## What are MutatingAdmissionPolicies?
+
+Mutating admission policies offer a declarative, in-process alternative to mutating admission webhooks.
+
+Mutating admission policies use the Common Expression Language (CEL) to declare mutations to resources.
+Mutations can be defined either with an *apply configuration* that is merged using the
+[server side apply merge strategy](/docs/reference/using-api/server-side-apply/#merge-strategy)
+or a [JSON patch](https://jsonpatch.com/).
+
+Mutating admission policies are highly configurable, enabling policy authors to define policies
+that can be parameterized and scoped to resources as needed by cluster administrators.
+
+## What resources make a policy
+
+A policy is generally made up of three resources:
+
+- The MutatingAdmissionPolicy describes the abstract logic of a policy
+  (think: "this policy sets a particular label to a particular value").
+
+- A MutatingAdmissionPolicyBinding links the above resources together and provides scoping.
+  If you only want to set an `owner` label for `Pods`, the binding is where you would
+  specify this mutation.
+
+- A parameter resource provides information to a MutatingAdmissionPolicy to make it a concrete
+  statement (think "set the `owner` label to something like `company.example.com`").
+  A native type such as ConfigMap or a CRD defines the schema of a parameter resource.
+  MutatingAdmissionPolicy objects specify what Kind they are expecting for their parameter resource.
+
+At least a MutatingAdmissionPolicy and a corresponding MutatingAdmissionPolicyBinding
+must be defined for a policy to have an effect.
+
+If a MutatingAdmissionPolicy does not need to be configured via parameters, simply leave
+`spec.paramKind` in  MutatingAdmissionPolicy not specified.
+
+## Getting Started with Mutating Admission Policy
+
+Mutating Admission Policy is part of the cluster control-plane. You should write
```

Comment:
```suggestion
Mutating admission policy is part of the cluster control-plane. You should write
```

Reply:
Fixed. Is the title fine or should that also be lower case?

`## Getting Started with Mutating Admission Policy`